### DISCUSSION TITLE
Is vercel banned in China?

### DISCUSSION THREAD

**Discussion by so1ve** [vercel/vercel#6719] [Help]
When I visit `vercel.com` and any websites pointing to `76.223.126.88` and `76.76.21.21`, the browser prompts "connection has been reset".

**Comment by leerob** [comment]
https://vercel.com/support/articles/how-do-i-prevent-my-vercel-deployment-from-being-blocked-in-china

**Reply by so1ve** [reply]
In fact, I have read the document. However, the current situation is that IP `76.223.126.88` and CNAME pointing to `cname-china.vercel-dns.com` cannot take effect. This is a recent situation.

**Reply by so1ve** [reply]
Maybe GFW caused this phenomenon.

**Reply by so1ve** [reply]
It's strange... Vercel can visit normally now.

**Reply by so1ve** [reply]
The current situation is sometimes accessible and sometimes not.

**Comment by eligao** [comment]
Still got a few disconnections here and there, you can probe with this : https://ping.sx/check-port?t=www.vercel.com

**Reply by eligao** [reply]
![image](https://user-images.githubusercontent.com/2323992/133456604-b2e3db10-2032-4629-b794-f4c7197db12b.png)

**Comment by Moto-Venturer** [comment]
Vercel is not blocked in China, but users in the country may face slow and unstable connections due to the Great Firewall. To avoid these issues, your best option is to host inside China. 

If you want a hassle free, CI/CD build-in service, Vercel gives you for China, concider 21YunBox instead. It like Vercel, but built for China.

https://www.21cloudbox.com/solutions/how-to-speed-up-vercel-in-china.html

**Reply by so1ve** [reply]
Yup 21yunbox is fast in Mainland China, however you have sent me TOO MANY EMAILS to ask me to use your service. I tried but there is only a 7-day free trial. Your marketing is SO FUCKING TERRIBLE that I will never use your service again. I would buy vercel's pro rather than your service.

**Comment by manueljenni** [comment]
Currently in Shanghai and the main page is blocked, as well as *.vercel.app domains.

**Reply by JacobSyndeo** [reply]
I was in Guangdong province (near Shenzhen) over the holidays, and my experience was the same.
